plugins/GSdx/GSState.h

```diff
@@ -243,6 +243,7 @@ class GSState : public GSAlignedClass<32>
 	void ResetHandlers();
 
 	int GetFramebufferHeight();
+	void SaturateOutputSize(GSVector4i*);
```

Comment:
Otherwise pass by const ref or value. And return the object. And please put a parameter name.

Pointers are error prone. You have no guarantee that `*r` is a `GSVector4i` object it could be anything.